### PR TITLE
feat: Add Mautic 4 and 5 to service templates

### PR DESCRIPTION
**Pull request by JuanxCursed**
Add Mautic 4 and 5 to service templates.
I hope is helpful to someone.

### CODE REVIEW

File: templates/compose/mautic5.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      - 'MAUTIC_DB_HOST=${MYSQL_HOST}'
+      - 'MAUTIC_DB_PORT=${MYSQL_PORT}'
+      - 'MAUTIC_DB_DATABASE=${MYSQL_DATABASE}'
+      - 'MAUTIC_DB_USER=${MYSQL_USER}'
+      - 'MAUTIC_DB_PASSWORD=${SERVICE_PASSWORD_64_MYSQL}'
+      - 'MAUTIC_MESSENGER_DSN_EMAIL=${MAUTIC_MESSENGER_DSN_EMAIL}'
+      - 'MAUTIC_MESSENGER_DSN_HIT=${MAUTIC_MESSENGER_DSN_HIT}'
+    depends_on:
+      mautic_web:
+        condition: service_health
```
Comment (Skeyelab) [inline templates/compose/mautic5.yaml:106]: There is a typo here: `service_health` should be `service_healthy`

Comment (Skeyelab) [reply templates/compose/mautic5.yaml:106]: Also, I think there is an issue with the version 5 config.  The web process cannot find the DB.

```2024-09-12T16:21:39.681752542Z mysqladmin: connect to server at '' failed
2024-09-12T16:21:39.683000375Z error: 'Can't connect to server on 'localhost' (111)'
2024-09-12T16:21:39.683031602Z Check that mariadbd is running and that the socket: '/run/mysqld/mysqld.sock' exists!```
Is appearing in the logs

Comment (Skeyelab) [reply templates/compose/mautic5.yaml:106]: @JuanxCursed ^

Comment (JuanxCursed) [reply templates/compose/mautic5.yaml]: @Skeyelab, thanks for the reporting, and sorry take that long to update the PR.

Fix it on commit 1b8365d

**Comment by peaklabs-dev** [comment]
Thank you for the PR 💜. I will test it and fix it if necessary.

**Comment by volfadar** [comment]
> Add Mautic 4 and 5 to service templates. I hope is helpful to someone.

Ohhh yes! Finally! I've been waiting so long. I was stuck on the Traefik configuration. Right now, I'm using Mautic with 'vanilla Docker' (lol). thanks a lot @JuanxCursed

**Pull request merged**